src/libmongoc/tests/mock_server/request.c

```diff
@@ -517,7 +519,7 @@ request_matches_kill_cursors (const request_t *request, int64_t cursor_id)
  *
  * request_matches_msg --
  *
- *       Test that a client OP_MSG matches a pattern. The OP_MSG consists
+ *       Test that client OP_MSGs match a pattern. The OP_MSGs consists
```

Comment:
I suggest reverting this comment.  This expects one OP_MSG. The OP_MSG may consist of a sequence of documents in multiple sections. See https://docs.mongodb.com/manual/reference/mongodb-wire-protocol/#op_msg.

Reply:
+1